docs/additional-functionality/rapids-shuffle.md

```diff
@@ -58,24 +61,22 @@ MLNX_OFED driver and `nv_peer_mem` kernel module.
    [file a GitHub issue](https://github.com/NVIDIA/spark-rapids/issues) so we can investigate 
    further.
     
-2. (skip if you followed Step 1) For setups without RoCE/Infiniband, UCX 1.9.0 packaging 
-   requires RDMA packages for installation. UCX 1.10.0+ will relax these requirements for these
-   types of environments, so this step should not be needed in the future.
+2. Fetch and install the UCX package for your OS and CUDA version 
+   [UCX 1.10.1](https://github.com/openucx/ucx/releases/tag/v1.10.1).
    
-   If you still want to install UCX 1.9.0 in a machine without RoCE/Infiniband hardware, please 
-   build and install `rdma-core`. You can use the [Docker sample below](#ucx-minimal-dockerfile) 
-   as reference.
-    
-3. Fetch and install the UCX package for your OS and CUDA version 
-   [UCX 1.9.0](https://github.com/openucx/ucx/releases/tag/v1.9.0).
+   UCX versions 1.10.1 requires the user to install `libnuma1`. RDMA packages have extra 
```

Comment:
Should we provide any instructions on installing libnuma1, or is it obvious enough that people need to an apt-get or yum install?  Will this requirement go away in a future version of UCX?

Reply:
It'll install as package dependency: https://github.com/openucx/ucx/blob/v1.10.x/debian/control.in#L17

Reply:
It'll be installed as package dependency: https://github.com/openucx/ucx/blob/v1.10.x/debian/control.in#L17

Reply:
@petro-rudenko is right. For example: `sudo apt install ./ucx-v1.10.1-ubuntu18.04-mofed5.x-cuda11.2.deb`

```
The following NEW packages will be installed:
  libnuma1 ucx
0 upgraded, 2 newly installed, 0 to remove and 420 not upgraded.
```

If you used `dpkg -i` the package is not automatically installed (but a `apt install -f` later does resolve). I'll remove this from the doc, as the package already specifies this.

Reply:
Changed our sample dockerfiles to use `apt install [ucx-package].deb`. I assume the same holds for CentOS, but we currently don't have a Dockerfile example for CentOS. I'll add, but maybe I should split these into dockerfiles rather than have them in the markdown.

Reply:
Ok trying a few things for CentOS, I'll add as a follow up issue and stick with Ubuntu (rdma-core is giving me trouble for CentOS).

Reply:
https://github.com/NVIDIA/spark-rapids/issues/2480